### DISCUSSION TITLE
Any way to rotate a marker?

### DISCUSSION THREAD

**Discussion by andydeforest** [vue-leaflet/vue-leaflet#197] [Q&A]
Wondering if it's possible to rotate a marker. I'm familiar with leaflet-rotated-marker but I'm not familiar enough to do my own implementation. Would I be able to rotate a marker with CSS maybe?

**Comment by bagasnur** [comment]
[optional] you can use this

https://github.com/littleimagine/vue-leaflet-rotate-marker
